modules/core/src/test/scala/org/scalasteward/core/update/PruningAlgTest.scala

```diff
@@ -192,7 +193,7 @@ class PruningAlgTest extends FunSuite {
             DependencyInfo("org.scala-lang".g % "scala-library".a % "2.12.14", List("build.sbt")),
             DependencyInfo("org.scala-lang".g % "scala-library".a % "2.13.5", List("build.sbt"))
           ),
-          List(MavenRepository("public", "https://repo5.org/maven/", None))
+          List(MavenRepository("public", "https://repo5.org/maven/", None, Nil))
```

Comment:
Some tests on this file requires updates too
```
==> X org.scalasteward.core.update.PruningAlgTest.needsAttention: group-specific frequency (monthly) vs repo frequency (asap)  0.003s scala.MatchError: Left(DecodingFailure(Attempt to decode value on failed cursor, List(DownField(headers), DownField(MavenRepository), DownArray, DownField(resolvers), DownArray, DownField(dependencyInfos)))) (of class scala.util.Left)
    at org.scalasteward.core.update.PruningAlgTest.$anonfun$new$11(PruningAlgTest.scala:220)
==> X org.scalasteward.core.update.PruningAlgTest.needsAttention: group-specific frequency (asap) vs repo frequency (monthly)  0.001s scala.MatchError: Left(DecodingFailure(Attempt to decode value on failed cursor, List(DownField(headers), DownField(MavenRepository), DownArray, DownField(resolvers), DownArray, DownField(dependencyInfos)))) (of class scala.util.Left)
    at org.scalasteward.core.update.PruningAlgTest.$anonfun$new$14(PruningAlgTest.scala:331)
==> X org.scalasteward.core.update.PruningAlgTest.needsAttention: dependency can have artifactId and version  0.0s scala.MatchError: Left(DecodingFailure(Attempt to decode value on failed cursor, List(DownField(headers), DownField(MavenRepository), DownArray, DownField(resolvers), DownArray, DownField(dependencyInfos)))) (of class scala.util.Left)
```

Reply:
I pushed a fixed version to the branch. My bad.

Thanks for having a look